NewHorizons/Builder/General/SectorBuilder.cs

```diff
@@ -22,7 +25,7 @@ public static Sector Make(GameObject planetBody, OWRigidbody owRigidBody, float
             sectorGO.AddComponent<OWTriggerVolume>();
 
             Sector S = sectorGO.AddComponent<Sector>();
-            S._name = (Sector.Name)24;
+            S._name = Name;
```

Comment:
this breaks OWO

Reply:
no it doesn't. OWO doesn't use (Sector.Name)24.